server/opts.go

```diff
@@ -828,6 +843,29 @@ func (o *Options) processConfigFileLine(k string, v interface{}, errors *[]error
 		o.TLSTimeout = tc.Timeout
 		o.TLSMap = tc.Map
 
+		// Need to keep track of path of the certs for OCSP Stapling.
+		o.TLSCert = tc.CertFile
+		o.TLSKey = tc.KeyFile
+		o.TLSCaCert = tc.CaFile
+	case "state_dir":
```

Comment:
can we use a different. name here? state is too generic a name.

Reply:
will probably need this just in case jetstream is not set

Reply:
Think maybe for this first pass we will remove this and always fetch staples on boot // avoid caching to address correctly store dir location, permissions, etc...